packages/nuxt-locokit/src/styles/theme.css

```diff
@@ -17,7 +17,7 @@
   --border-color: #ced4da;
   --table-border-color: #e9ecef;
   --border: 1px solid var(--border-color);
-  --border-radius: 2px;
+  --border-radius: 4px;
```

Comment:
I see you :-)

Reply:
Yep, in client's projects this variable is always 4px. And I like it! :heart: